modules/react/popup/lib/PopupCloseIcon.tsx

```diff
@@ -1,30 +1,29 @@
 import React from 'react';
 
 import {xIcon} from '@workday/canvas-system-icons-web';
-import {
-  styled,
-  createSubcomponent,
-  StyledType,
-  ExtractProps,
-} from '@workday/canvas-kit-react/common';
+import {createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
 import {TertiaryButton} from '@workday/canvas-kit-react/button';
+import {space} from '@workday/canvas-kit-react/tokens';
 
 import {usePopupCloseButton, usePopupModel} from './hooks';
 
-const closeIconSpacing = 4;
-
 export interface PopupCloseIconProps extends ExtractProps<typeof TertiaryButton, never> {}
 
-const StyledCloseIcon = styled(TertiaryButton)<StyledType & PopupCloseIconProps>({
-  position: 'absolute',
-  right: closeIconSpacing,
-  top: closeIconSpacing,
-});
-
 export const PopupCloseIcon = createSubcomponent('button')({
   displayName: 'Popup.CloseIcon',
   modelHook: usePopupModel,
   elemPropsHook: usePopupCloseButton,
 })<PopupCloseIconProps>(({children, ...elemProps}, Element) => {
-  return <StyledCloseIcon as={Element} size="medium" icon={xIcon} type="button" {...elemProps} />;
+  return (
+    <TertiaryButton
+      as={Element}
+      size="medium"
+      icon={xIcon}
+      type="button"
+      position="absolute"
+      right={space.xxxs}
+      top={space.xxxs}
```

Comment:
Q: Does position props support token values as `right="xxxs"`. If it doesn't, should it support?

Reply:
It currently does not. I'd love for all of out style props to be aware of our tokens like this!